packages/kbn-monaco/src/painless/antlr/painless_lexer.ts

```diff
@@ -184,13 +185,9 @@ export class painless_lexer extends Lexer {
 	// @Override
 	public sempred(_localctx: RuleContext, ruleIndex: number, predIndex: number): boolean {
 		switch (ruleIndex) {
-    // DO NOT CHANGE
-    // This is a manual fix to handle slashes appropriately
 		case 31:
```

Comment:
I think we need to add this comment back in, and change it to `32` based on the addition of `DOLLAR`. Otherwise, the slash is broken again.

Long term, it would be nice to figure out the root of this problem 😄  (briefly explained in the [readme](https://github.com/elastic/kibana/tree/main/packages/kbn-monaco/src/painless#compiling-antlr)). 

![Painless-Lab-Dev-Tools-Elastic](https://user-images.githubusercontent.com/5226211/154342134-0dc82702-0602-4f47-a569-0c7dbb92094c.png)

Reply:
Nice find.  Updated.
![image](https://user-images.githubusercontent.com/24232742/154353252-d07475b8-4505-40d5-9157-cf47da4825a3.png)